### PR TITLE
1. fix all import-modules

### PR DESCRIPTION
**Pull request by TingluoHuang**
2. pump up task version

### CODE REVIEW

File: Tasks/DeployAzureResourceGroup/Utility.ps1 [hunk 1]
```diff
@@ -18,7 +18,8 @@ function Get-SingleFile($files, $pattern)
 function Get-File($pattern)
 {
     #Find the File based on pattern
-    
+    import-module "Microsoft.TeamFoundation.DistributedTask.Task.Common"
```
Comment (abhinitmodi) [inline Tasks/DeployAzureResourceGroup/Utility.ps1]: Why are we importing dlls inside the helper funtions of the task. If we import it once in the main script, i.e. DeployAzureResourceGroup in this case would not that be sufficient?

File: Tasks/DeployAzureResourceGroup/DtlServiceHelper.ps1 [hunk 2]
```diff
@@ -199,6 +204,7 @@ function Check-EnvironmentNameAvailability
     {
         Write-Verbose -Verbose "Checking machine group name availability"
 
+        import-module "Microsoft.TeamFoundation.DistributedTask.Task.DevTestLabs"
```
Comment (abhinitmodi) [inline Tasks/DeployAzureResourceGroup/DtlServiceHelper.ps1]: Silimarly here. DevTestLabs dll is anyway imported in the main ps1 file, why import again.

File: Tasks/AzureWebPowerShellDeployment/Publish-AzureWebDeployment.ps1 [hunk 3]
```diff
@@ -41,6 +41,8 @@ Write-Host "Package= $Package"
 Write-Host "AdditionalArguments= $AdditionalArguments"
 
 #Find the package to deploy
+import-module "Microsoft.TeamFoundation.DistributedTask.Task.Common"
```
Comment (pryenugu) [inline Tasks/AzureWebPowerShellDeployment/Publish-AzureWebDeployment.ps1]: this task was working without the import specifically. Is it because its path was added to the PSModule path? Why was this needed post your change?

File: Tasks/CopyFilesToMachines/CopyFilesHelper.ps1 [hunk 4]
```diff
@@ -11,7 +11,8 @@ function DoComplete-ResourceOperation
     $logs = New-Object 'System.Collections.Generic.List[System.Object]'         
     $resourceOperationLog = New-OperationLog -Content $log
     $logs.Add($resourceOperationLog)
-
+    
+    import-module "Microsoft.TeamFoundation.DistributedTask.Task.DevTestLabs"
```
Comment (pryenugu) [inline Tasks/CopyFilesToMachines/CopyFilesHelper.ps1]: We are importing this module in the CopyFileToMachines.ps1 and then executing this function in this script which invokes cmdlet Complete-ResourceOperation. I think this would not be needed here as it is already being imported in the script which calls it.

File: Tasks/CopyFilesToMachines/CopyJob.ps1 [hunk 5]
```diff
@@ -19,20 +19,20 @@ param (
...
-	{
+   if($cleanTargetBeforeCopy -eq "true")
+    {
 		$cleanTargetPathOption = '-CleanTargetPath'
-	}
+    }
 
     Write-Verbose "Initiating copy on $fqdn " -Verbose
-   
+    import-module "Microsoft.TeamFoundation.DistributedTask.Task.DevTestLabs"
```
Comment (pryenugu) [inline Tasks/CopyFilesToMachines/CopyJob.ps1]: Similar to the above comment, we might not need to import again.

File: Tasks/AzureWebPowerShellDeployment/Publish-AzureWebDeployment.ps1 (hunk 3 above)
Comment (TingluoHuang) [inline Tasks/AzureWebPowerShellDeployment/Publish-AzureWebDeployment.ps1]: the task was working because a previous task import the task.common module, it cached in memory, so this task works fine, but if this task was the one that run first, it will cause problem.

File: Tasks/CopyFilesToMachines/CopyFilesHelper.ps1 (hunk 4 above)
Comment (TingluoHuang) [inline Tasks/CopyFilesToMachines/CopyFilesHelper.ps1]: will remove.
